<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>borrowBookEditView</class>
 <widget class="QWidget" name="borrowBookEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="bookId">
         <property name="text">
          <string>书本ID：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="readerId">
         <property name="text">
          <string>读者ID：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="bookIdInput"/>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QLabel" name="bookName">
         <property name="text">
          <string>书名：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="dueTime">
         <property name="text">
          <string>到期时间：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="bookNameInput"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="readerName">
         <property name="text">
          <string>借阅人姓名：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" rowspan="2">
        <widget class="QLineEdit" name="readerNameInput"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="readerIdInput">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="borrowTime">
         <property name="text">
          <string>借阅时间：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDateEdit" name="borrowTimeInput"/>
       </item>
       <item row="6" column="1">
        <widget class="QDateEdit" name="dueTimeInput"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="btCancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btBorrow">
         <property name="text">
          <string>借书</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
